lib/schema_dev/github_actions.rb

```diff
@@ -0,0 +1,264 @@
+require 'pathname'
+require 'yaml'
+
+require_relative 'gemfile_selector'
+
+module SchemaDev
+  module GithubActions
+    extend self
+
+    WORKFLOW_FILE = ".github/workflows/prs.yml"
+
+    HEADER = <<~YAML
+      # This file was auto-generated by the schema_dev tool, based on the data in
+      #                 ./schema_dev.yml
+      # Please do not edit this file; any changes will be overwritten next time
+      # schema_dev gets run.
+    YAML
+
+    BASIC_WORKFLOW = {
+      name:        'CI PR Builds',
+      on:          {
+        push:         {
+          branches: %w[master],
+        },
+        pull_request: nil,
+      },
+      concurrency: {
+        group:                '${{ github.head_ref }}',
```

Comment:
In the generated YAML, it comes out quoted for some reason -- see [this failing run](https://github.com/teamsimplepay/schema_plus_core/actions/runs/1590681767/workflow). If you look at this PR, it works fine when unquoted -- not sure how we can force that? We might need to migrate to a text-template like approach :man_shrugging: